<launch>

    <arg name="num_robots" default="4"/>
    <arg name = "links" default="[[0,1],[0,2],[0,3],[1,2],[1,3],[2,3]]"/>

    <param name = "inter_distance_x" value="1"/>
    <param name = "inter_distance_y" value="0"/>

    <node pkg="mr_rendezvous_deployment" type="deployment_Laplacian.py" name="deploy" args="$(arg num_robots) $(arg links) True"  output="screen">
        <param name = "t_local" value = "0.01"/>
    </node>

    <node pkg="mr_rendezvous_deployment" type="visualization_node.py" name="position_plotter" args="$(arg num_robots)"></node>
</launch>
